CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased
 
+## v3.1.2 - 2020-09-24
+
+- Added Field_Group class
+- Improvements to Repeater and Block_Config
```

Comment:
These should go in the "Unreleased" section, since they haven't been released. The version number and date will be added when we run the the release script.